realm/realm-library/src/main/java/io/realm/ImportFlag.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm;
+
+import io.realm.annotations.Beta;
+
+/**
+ * This class describe how data is saved to Realm when saving whole objects.
+ *
+ * @see Realm#copyToRealm(RealmModel, ImportFlag...)
+ */
+@Beta
+public enum ImportFlag {
+
+    /**
+     * With this flag enabled, fields will not be written to the Realm file if they contain the same
+     * value as the value already present in the Realm.
+     * <p>
+     * For local Realms this only has an impact on change listeners which will not report changes to
+     * those fields that was not written.
+     * <p>
+     * For synchronized Realms this also impacts the server, which will see improved performance as
+     * there is less changes to upload and merge into the server Realm.
+     * <p>
+     * It also impact how the server merges changes from different devices. Realm uses a
+     * last-write-wins approach when merging individual fields in an object, so if a field is not
```

Comment:
it could be nice to point to the blog post Simon wrote to explain the merge algorithm

Reply:
Good idea. Done